aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -609,30 +609,6 @@ void THCTensor_(cfmod)(THCState *state, THCTensor *self, THCTensor *src1, THCTen
   }
 }
 
-void THCTensor_(addcmul)(THCState *state, THCTensor *self_, THCTensor *t, scalar_t value, THCTensor *src1, THCTensor *src2)
-{
-  THCAssertSameGPU(THCTensor_(checkGPU)(state, 4, self_, t, src1, src2));
-  if(self_ != t)
-  {
-    THCTensor_(resizeAs)(state, self_, t);
-    THCTensor_(copy)(state, self_, t);
-  }
-  else
-  {
-    THArgCheck(THCTensor_(nElement)(state, self_) == THCTensor_(nElement)(state, src1),
-               1, "sizes do not match");
-  }
-
-  THArgCheck(THCTensor_(nElement)(state, src1) == THCTensor_(nElement)(state, src2),
-             3, "sizes do not match");
-
-  if (!THC_pointwiseApply3<scalar_t, scalar_t, scalar_t>(state, self_, src1, src2, TensorAddCMulOp<scalar_t>(value))) {
```

Comment:
Please also remove `TensorAddCMulOp` from the code base.

Reply:
Removed